lib/sensitive_data_filter/middleware/parameter_parser.rb

```diff
@@ -27,10 +27,14 @@ def can_parse?(content_type)
 
       def parse(params)
         @parse.call params
+      rescue
```

Comment:
It is important we catch only the expected parse errors and let everything else explode as it should.